composer.json

```diff
@@ -26,7 +26,8 @@
         "drupal/swiftmailer": "^1.0@beta",
         "drupal/rules": "^3.0@alpha",
         "drupal/autologout": "^1.0",
-        "drupal/file_entity": "^2.0@beta"
+        "drupal/file_entity": "^2.0@beta",
+	"drupal/clamav": "^1.0"
```

Comment:
Hmm, what is this commit for? We're not actually using the `clamav` module itself, right?

Reply:
Correct, it was deleted in a subsequent commit.